src/test.js

```diff
@@ -208,7 +208,7 @@ Test.prototype = {
 
 		const runHook = () => {
 			if ( hookName === "before" ) {
-				if ( hookOwner.unskippedTestsRun !== 0 ) {
+				if ( hookOwner.testsRun !== 0 ) {
```

Comment:
I'm wrapping my head around the bug now and want to make sure I understand each part of the change.

I believe this line is not a change in behaviour but an update to keep the same behaviour, meaning that the value we used to keep in `unskippedTestsRun` is now effectively kept in `testsRun`, only incremented for tests that were "really" run. Is that right?

Reply:
Yep, thats the correct understanding!
And to make the reverse clear, f tests that were not run(eg. skip, or tests where `valid()` returns false) would be kept using `testsIgnored`